### PR TITLE
EUI-8524 QQ create component step2

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8524

### Change description ###
Create QQ component - Step 2

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: src/cases/containers/query-management-container/query-management-container.component.spec.ts [hunk 1]
```diff
@@ -94,20 +104,31 @@ describe('QueryManagementContainerComponent', () => {
...
       expect(component.queryItem).toBeUndefined();
     });
 
-    it('should have the ccd-query-write-raise-query component', () => {
+    it('should set the queryCreateContext to be query item type of new query qualifying question options', () => {
+      expect(component.queryCreateContext).toEqual(QueryItemType.NEW_QUERY_QUALIFYING_QUESTION_OPTIONS);
+    });
+
+    it('should have the ccd-qualifying-question-options component', () => {
+      console.log('SUMMARY', component.showSummary);
```
Comment (Daniel-Lam) [inline src/cases/containers/query-management-container/query-management-container.component.spec.ts:123]: @johnbenjamin-hmcts - can remove this `console.log` statement

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - just a minor one

File: src/cases/containers/query-management-container/query-management-container.component.spec.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/cases/containers/query-management-container/query-management-container.component.spec.ts]: Thank you @Daniel-Lam, I have updated the code now.

**Review by Daniel-Lam — APPROVED** [review]

**Pull request merged**